packages/composables/src/hooks/useProduct/SwHelpers/getMainImageUrl.ts

```diff
@@ -0,0 +1,3 @@
+export default function getMainImageUrl(product, defaultUrl= "/img/product_thumb.png"){
```

Comment:
```suggestion
export default function getMainImageUrl(product){
```
it would be really painful to always provide fallback here, maybe later we would add this to config and then use here :)